docs/index.html

```diff
@@ -41,28 +41,30 @@ <h3>Supported data sources</h3>
             <div class="col-xs-2 text-right"><i class="fa fa-2x fa-code"></i></div>
             <div class="col-xs-10">
               <p>Source Code Management</p>
-              <p class="text-justify small">Git, <em>Baazar(*), Mercurial(*) (*exporting to git)</em></p>
+              <p class="text-justify small">Git, <i>Baazar(*), Mercurial(*) (* by exporting to git)</i></p>
             </div>
           </div>
           <div class="row">
             <div class="col-xs-2 text-right"><i class="fa fa-2x fa-bug"></i></div>
             <div class="col-xs-10">
               <p>Issues / Tasks Management</p>
-              <p class="text-justify small">Bugzilla, GitHub Issues, GitLab Issues, JIRA, Maniphest, Redmine</p>
+              <p class="text-justify small">Bugzilla, GitHub Issues (including comments), GitHub project boards events, 
```

Comment:
GitHub project boards events should be marked with `(*)`, since there is no dashboard them ATM

Reply:
Fix coming

Reply:
perfect thanks! Squash the commits into one once done and we are ready to go

Reply:
I think I've done it right. Let me know if it's ok